app/src/main/java/yt/javi/nfltweets/view/holders/TeamViewHolder.kt

```diff
@@ -0,0 +1,34 @@
+package yt.javi.nfltweets.view.holders
+
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import yt.javi.nfltweets.R
+import yt.javi.nfltweets.domain.model.Team
+
+
+class TeamViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+    private val teamName = view.findViewById<TextView>(R.id.team_name)
```

Comment:
Check ButterKnife or android extensions for kotlin. It will make your life easier.

Reply:
Great, I will change it :)